recipe/doc/recipe_guide.md

```diff
@@ -69,6 +73,53 @@ This package is split into the `cuda-nvcc` package -- which is architecture spec
 x86-64 Linux) -- and the `cuda-nvcc_${TARGET_PLATFORM}` packages -- each of which is architecture-independent and may be installed on any target, but are only suitable for use in compiling code for the specified target platform.
 This approach allows using host machines with a single platform to compile code for multiple platforms.
 
+## Building for ARM Tegra devices
+
+> [!IMPORTANT]
+> Support for Tegra builds on conda-forge is only available for CUDA 12.9 and later.
+
+The [arm-variant](https://github.com/conda-forge/arm-variant-feedstock) package allows
+end-users to select which variant of ARM packages are installed into their environment.
+However, since there are no Tegra devices available for compilation, these packages must be
+cross-compiled from x86. Recipes that wish to build for both SBSA ARM and Tegra ARM devices
+should have something like the following in their recipe:
+
+```yaml
+requirements:
+  build:
+    - {{ compiler('cuda') }}
+    - arm-variant * {{ arm_variant_type }}
+  host:
+    - arm-variant * {{ arm_variant_type }}
```

Comment:
```suggestion
requirements:
  build:
    - {{ compiler('cuda') }}
    - arm-variant * {{ arm_variant_type }}  # [linux and aarch64]
```
Since I first drafted these docs, we have added strong run_exports to `arm-variant`, so it should be sufficient to just add `arm-variant` to the build environment. Then the host and run environments will also be constrained to the same `arm-variant`.